app/scripts/modules/oracle/pipeline/stages/bake/bakeStage.js

```diff
@@ -0,0 +1,207 @@
+'use strict';
+
+const angular = require('angular');
+
+import _ from 'lodash';
+
+import {
+  PIPELINE_CONFIG_PROVIDER,
+  BAKERY_SERVICE,
+  ACCOUNT_SERVICE,
+  NETWORK_READ_SERVICE,
+  SUBNET_READ_SERVICE
+} from '@spinnaker/core';
+
+module.exports = angular.module('spinnaker.oraclebmcs.pipeline.stage.bakeStage', [
+  require('./bakeExecutionDetails.controller.js'),
+  PIPELINE_CONFIG_PROVIDER,
+  BAKERY_SERVICE,
+  ACCOUNT_SERVICE,
+  NETWORK_READ_SERVICE,
+  SUBNET_READ_SERVICE
+])
+  .config(function(pipelineConfigProvider) {
+    pipelineConfigProvider.registerStage({
+      provides: 'bake',
+      cloudProvider: 'oraclebmcs',
+      label: 'Bake',
+      description: 'Bakes an image in the specified region',
+      templateUrl: require('./bakeStage.html'),
+      executionDetailsUrl: require('./bakeExecutionDetails.html'),
+      executionLabelTemplateUrl: require('core/pipeline/config/stages/bake/BakeExecutionLabel'),
+      defaultTimeoutMs: 60 * 60 * 1000, // 60 minutes
+      validators: [
+        {type: 'requiredField', fieldName: 'baseOs', preventSave: true},
```

Comment:
Using `preventSave` here is inconsistent with the other providers, but that's up to you.